### DISCUSSION TITLE
Add IssueNavigationLink git log and icon for idea

### DISCUSSION THREAD

**Discussion by CodePrometheus** [apache/skywalking#12194] [Ideas]
### What

1. This enables GitHub hyperlinks in IDEA for commits containing PR information.
Clicking on the link will directly open the corresponding GitHub PR.
The difference is as follows:
<img width="1120" alt="image" src="https://github.com/apache/skywalking/assets/66550292/43b05ce3-76aa-4f6e-9d87-35044f8fc80d">

Before:
<img width="1126" alt="image" src="https://github.com/apache/skywalking/assets/66550292/c3d59954-2a4e-40f4-add8-9b4e127cab4f">

2. Add icon for idea
<img width="751" alt="image" src="https://github.com/apache/skywalking/assets/66550292/4b14b806-ee46-428a-85b0-13b0c68444c7">

<img width="226" alt="image" src="https://github.com/apache/skywalking/assets/66550292/8dd5df58-02e0-479f-ab6b-e6cee8b37753">

### How

- change .gitignore
- adde vcs.xml and icon.png in `.idea`

### Are you willing to submit a pull request to implement this on your own?
- [x] Yes I am willing to submit a pull request on my own!

### Code of Conduct
- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
<1> should be helpful. I am not sure whether adding .idea could cause local conflicts, because many developers use import to setup project by default. Meanwhile, once we added them, do we introduce local file conflicts?

**Reply by CodePrometheus** [reply]
I just recompiled it locally and tried it, If it is imported by default, the main new points are as follows `component`, when the user does not have a clone submodule, will be reminded by idea, after the compilation is completed, `vcs.xml` will not be changed.
<img width="1154" alt="image" src="https://github.com/apache/skywalking/assets/66550292/fca7aba3-25bb-4908-95eb-cac88edfafba">
